README.md

```diff
@@ -9,7 +9,7 @@ High performance routing engine written in C++14 designed to run on OpenStreetMa
 The following services are available via HTTP API, C++ library interface and NodeJs wrapper:
 - Nearest - Snaps coordinates to the street network and returns the nearest matches
 - Route - Finds the fastest route between coordinates
-- Table - Computes the duration of the fastest route between all pairs of supplied coordinates
+- Table - Computes the duration of the fastest route between all pairs of supplied coordinates, and optionally returns distances of these routes.
```

Comment:
Maybe just shorten this to 
`- Table - Computes the duration or distances of the fastest route between all pairs of supplied coordinates`

The other services don't list all their options either.